### ISSUE TITLE
Creating a new address should not require gateway and netmask

### ISSUE THREAD

**Issue by discordianfish** [tumblr/collins#476]
When creating a new specific IP via the POST endpoint, it requires to specify gw, netmask and pool beside the address. This seems redundant since the gateway and netmask can be looked up in the pool: https://tumblr.github.io/collins/api.html#api-ipam-update-an-address

Maybe I miss something but I can't see any reason this is required from the user instead of collins looking that up in the configured pools.

**Comment by byxorna** [comment]
@discordianfish I agree. Would you be interested in making a diff to fix this? (if omitted, determine from pool config, otherwise use user supplied vals?).

**Comment by discordianfish** [comment]
Same here, not doing anything on bare metal right now. If I end up doing something collins base in the future, I'm happy to pick that up again though.
